angular2/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "typescript-angular2",
+    "version": "1.0.0",
+    "description": "Angular 2/TypeScript demo",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/TypeScriptSamples.git"

```

Comment:
I would hold off on adding this sample untill this version is shipped. We do not want confuse ppl about what works where.